src/features/DebugSession.ts

```diff
@@ -80,6 +80,12 @@ export class DebugSessionFeature implements IFeature {
                     }
                 }
             }
+            else if (config.script) {
+                // In this case, the user has explicitly defined a script path
+                // so make sure to set the cwd to that path if the cwd wasn't
+                // explicitly set
+                config.cwd = config.cwd || config.script;
```

Comment:
When I was testing this, it seemed like $HOME was being used by default when we didn't try to set the cwd ourselves.  I tried using $HOME and the path came out like `c:\Users\daviwil\$HOME` for some reason, so I just went back to sending it as `undefined`